app/uk/gov/hmrc/tradergoodsprofilesrouter/models/request/MaintainProfileRequest.scala

```diff
@@ -37,11 +37,11 @@ object MaintainProfileRequest {
       (JsPath \ "ukimsNumber").read(lengthBetween(32, 32)) and
       (JsPath \ "nirmsNumber").readNullable(lengthBetween(13, 13)) and
       (JsPath \ "niphlNumber")
-        .readNullable(lengthBetween(8, 8)))(MaintainProfileRequest.apply _)
+        .readNullable(validNiphls))(MaintainProfileRequest.apply _)
 
   implicit lazy val writes: OWrites[MaintainProfileRequest] =
     ((JsPath \ "actorId").write[String] and
       (JsPath \ "ukimsNumber").write[String] and
       (JsPath \ "nirmsNumber").writeNullable[String] and
-      (JsPath \ "niphlsNumber").writeNullable[String])(unlift(MaintainProfileRequest.unapply))
+      (JsPath \ "niphlNumber").writeNullable[String])(unlift(MaintainProfileRequest.unapply))
```

Comment:
can we keep the name consistent everywhere if its going to be `niphlNumber ` and not `niphlsNumber ` ?

Reply:
The correct one is `niphlNumber` as updated here. It is updated everywhere else or is there anyone missing?